package.json

```
@@ -4,7 +4,7 @@
   "version": "1.47.1",
   "description": "A framework to support application development with Terra components",
   "engines": {
-    "node": ">=8.9.2 <11"
+    "node": ">=10 <16"
```

lets try less than 15, npm 7 doesn't much care for our peer dependency issues.
```suggestion
    "node": ">=10 <15"
```